src/Dfe.EarlyYearsQualification.Content/Entities/PhaseBanner.cs

```diff
@@ -0,0 +1,14 @@
+using Contentful.Core.Models;
+
+namespace Dfe.EarlyYearsQualification.Content.Entities;
+
+public class PhaseBanner
+{
+  public string PhaseName { get; init; } = string.Empty;
+
+  public Document? Content { get; set; }
```

Comment:
suggestion: can be `init` (don't want code to be able to change this value)

Reply:
Fixed now